libevmasm/ConstantOptimiser.h

```diff
@@ -141,6 +146,8 @@ class ComputeMethod: public ConstantOptimisationMethod
 protected:
 	/// Tries to recursively find a way to compute @a _value.
 	AssemblyItems findRepresentation(u256 const& _value);
+	/// Recomputes the value from the representation and checks for correctness.
+	bool checkRepresentation(u256 const&);
```

Comment:
For clarity it may make sense to actually have `AssemblyItems` as a parameter here and not use the member directly.